_posts/2015-03-18-Wifi-pairing.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Wifi pairing
+---
+Earlier I suggested that the Known groups for Wifi Direct could be teached for the device.
+Thus making the Wifi direct "pairable" same way as Bluetooth.
```

Comment:
Yep, did that on hurry, thus was not thinking how it sounds like. Yours sounds lot better